functions/pupil-status/pupil-status-analysis.service.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const R = require('ramda')
+
+/**
+ *
+ * @param data
+ * @return {string}
+ *
+ * Data Format
+ [ { pupil_id: 12,
+   pupilStatusCode: 'UNALLOC',
+   check_id: 1,
+   checkStatusCode: 'NEW' }
+ ]
+ */
+
+module.exports = {
+  analysePupilData: function analysePupilData (data) {
```

Comment:
@jon-shipley would it be possible to provide a jsdoc description about this method?

Reply:
Good spot!  JSDoc improved.